index.html

```diff
@@ -323,8 +323,16 @@
 
                     this.refreshBuilds();
                     },
-                watch: { stream: 'refreshBuilds' },
+                watch: {
+                    stream() {
```

Comment:
Shouldn't this be `stream: function() { ...`? Or is this a Vue sugar syntax thing which allows writing it that way too?

Reply:
It comes out when I was looking for "invoke multiple functions on watch function ": https://stackoverflow.com/questions/55177967/call-two-methods-on-watch-function-in-vuejs